src/user/models/create-user.dto.ts

```diff
@@ -0,0 +1,13 @@
+import { IsEmail, IsString } from 'class-validator';
+import { IsPassword } from 'src/auth/utils/validation-utils';
```

Comment:
Usually I do not add imports myself. It's auto imported like this and sometimes in relative form. Not sure what ts/eslint options control this. Does using these ts configuration make sense in BND?
```
"baseUrl": "./",
"paths": {
      "@ECommerce/*": ["src/*"]
},
```

Then writing imports like this `import { IsPassword } from '@ECommerce/auth/utils/validation-utils';`?

Reply:
by relative passes I mean typing the path relative to the current path, for example it will start with `../` if the required file is in the parent folder